shoplist/src/main/res/layout/main_fragment.xml

```diff
@@ -11,7 +11,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:padding="8dp"
-        android:text="Check out your shopping lists!"
```

Comment:
i prefer the text to be accessed from strings file for maintenance so change it would be great